classes/class-base.php

```diff
@@ -144,6 +144,18 @@ public function get_remote_server_root_url() {
 			: 'https://progressplanner.com';
 	}
 
+	/**
+	 * Get the placeholder image URL.
+	 *
+	 * @return array<string, string>
+	 */
+	public function get_placeholder_image_url() {
+		return [
+			'default' => 'https://placehold.co/1200x675',
```

Comment:
Hmm this seems like a good idea, but not really from a GDPR perspective 😅  can we replicate this easily in WP or otherwise?

Reply:
Let's just do this?

```php
function get_placeholder_svg( $width, $height ) {
	return 'data:image/svg+xml;base64,' . base64_encode( sprintf( '<svg width="%1$d" height="%2$d" xmlns="http://www.w3.org/2000/svg"><rect x="2" y="2" width="%3$d" height="%4$d" style="fill:#F6F5FB;stroke:#534786;stroke-width:2"/><text x="50%%" y="50%%" font-size="20" text-anchor="middle" alignment-baseline="middle" font-family="monospace" fill="#534786">%1$d&#215;%2$d</text></svg>', $width, $height, ( $width - 4 ), ( $height - 4 ) ) );
}
```